### PR TITLE
Adds Python (Django) Web App with PostgreSQL via ACA

### PR DESCRIPTION
**Pull request by kjaymiller**
Adds Python (Django) Web App with PostgreSQL via ACA

Steps to test:
- Deployed Locally using

```shell
cd website
npm install
npx docusaurus serve
```

adds: #118

### CODE REVIEW

**Review by savannahostrowski — APPROVED** [review]

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -226,6 +226,16 @@ const Users: User[] = [
...
   },
+  {
+    title: 'Python (Django) Web App with PostgreSQL via Azure Container Apps',
+    description:
+      'A space tourism web app that uses PostgreSQL for the database. The web app is containerize and deploy with Azure Container Apps with secrets managed by Azure Key Vault',
+      preview: require('./images/simple-flask-api-container.png'),
+    website: 'https://github.com/Azure-Samples',
+    author: 'Azure Content Team',
+    source: 'https://github.com/Azure-Samples/azure-django-postgres-aca',
+    tags: ['python','django', 'aca', 'azuredb-postgreSQL'],
```
Comment (hemarina) [inline website/src/data/users.tsx]: Do we want add 'bicep' tag as well?

File: website/src/data/users.tsx [hunk 2]
```diff
@@ -226,6 +226,16 @@ const Users: User[] = [
     source: 'https://github.com/Azure-Samples/msdocs-django-postgresql-sample-app',
     tags: ['python','django', 'webapps', 'azuredb-postgreSQL', 'appservice','vnets'],
   },
+  {
+    title: 'Python (Django) Web App with PostgreSQL via Azure Container Apps',
+    description:
+      'A space tourism web app that uses PostgreSQL for the database. The web app is containerize and deploy with Azure Container Apps with secrets managed by Azure Key Vault',
+      preview: require('./images/simple-flask-api-container.png'),
```
Comment (hemarina) [inline website/src/data/users.tsx]: I saw there is a minor difference with the image you post in the issue and ` simple-flask-api-container.png`, feel free to use save your image and use here.

File: website/src/data/users.tsx (hunk 1 above)
Comment (kjaymiller) [reply website/src/data/users.tsx]: I have the changes locally but don't think I can update this branch as I pulled from the wrong origin (I forked from @hemarina and not azure). Happy to adjust but I think if I did I would have to redo the entire PR.

File: website/src/data/users.tsx (hunk 2 above)
Comment (kjaymiller) [reply website/src/data/users.tsx]: I have the changes locally but don't think I can update this branch as I pulled from the wrong origin (I forked from @hemarina and not azure). Happy to adjust but I think if I did I would have to redo the entire PR.

**Comment by kjaymiller** [comment]
@hemarina - Looks Good happy to have this added!

**Pull request merged**